staging/src/k8s.io/apiserver/pkg/storage/interfaces.go

```diff
@@ -160,7 +161,10 @@ type Interface interface {
 	// (e.g. reconnecting without missing any updates).
 	// If resource version is "0", this interface will get current object at given key
 	// and send it in an "ADDED" event, before watch starts.
-	Watch(ctx context.Context, key string, resourceVersion string, p SelectionPredicate) (watch.Interface, error)
+	// The 'timeout' indicates the duration in which the client keeps watch. The server
```

Comment:
This sentence is misleading - it's the time after which server (not client) will close it, right?